test/e2e/apimachinery/resource_quota.go

```diff
@@ -335,6 +335,7 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		ginkgo.By("Creating a ResourceQuota")
 		quotaName := "test-quota"
 		resourceQuota := newTestResourceQuota(quotaName)
+		resourceQuota.Spec.Hard[v1.ResourceConfigMaps] = resource.MustParse(hardConfigMaps)
 		_, err = createResourceQuota(f.ClientSet, f.Namespace.Name, resourceQuota)
 		framework.ExpectNoError(err)
 
```

Comment:
drop the "we expect there to be two configmaps" comment block below. that is not a generally correct assumption when there exist clusters that inject configmaps by default (as the check for the "default" number of configmaps in the namespace at the beginning of the test shows)